docs/antora.yml

```diff
@@ -0,0 +1,23 @@
+name: asciidoctorj
+title: Asciidoctorj
+version: '2.4'
+asciidoc:
+  attributes:
+    release-version: 2.4.2
+    artifact-version: 2.4.2
+    asciidoctorj-epub3-version: 1.5.0-alpha.19
+    asciidoctorj-pdf-version: 1.5.3
+    jruby-version: 9.2.13.0
+    uri-jruby: http://jruby.org
```

Comment:
Please use url- as a prefix for URLs instead of uri-. The What WG now recommends using URL in favor of URI to avoid confusion.

Reply:
I am not modifying the README for now, at least, not untill we have the site.

Reply:
Understood.